docs/CHANGELOG-v1.md

```diff
@@ -106,7 +106,11 @@ What's changed since pre-release v1.19.0-B0010:
     [#1582](https://github.com/Azure/PSRule.Rules.Azure/issues/1582)
   - Fixed handling of storage accounts sub-resources with CMK by @BernieWhite.
     [#1575](https://github.com/Azure/PSRule.Rules.Azure/issues/1575)
-
+- New rules:
+  - Azure Resources:
+    - Check that nested deployments securely pass through administrator usernames @ms-sambell.
+      [#1479](https://github.com/Azure/PSRule.Rules.Azure/issues/1479)
+  
```

Comment:
@ms-sambell Just need to move this up to unreleased, since there has been a few releases.